docs/manual/must-call-checker.tex

```diff
@@ -0,0 +1,204 @@
+\htmlhr
+\chapterAndLabel{Must Call Checker}{must-call-checker}
+
+The Must Call Checker conservatively over-approximates
+the set of methods that an object should call before it is de-allocated.
+The checker does not enforce that the methods are called before objects are de-allocated,
+nor is the are its annotations refined after a must-call method is called; rather,
```

Comment:
I think ` is the` needs to be removed?

Otherwise, this paragraph is much clearer; thanks!